assets/src/apps/authoring/components/RightMenu/RightMenu.tsx

```diff
@@ -1,19 +1,21 @@
 import { JSONSchema7 } from 'json-schema';
 import { debounce } from 'lodash';
 import React, { useCallback, useState } from 'react';
-import { Accordion, Tab, Tabs } from 'react-bootstrap';
+import { Tab, Tabs } from 'react-bootstrap';
 import { useDispatch, useSelector } from 'react-redux';
-import { selectCurrentActivity } from '../../../delivery/store/features/activities/slice';
+import { selectCurrentActivity, upsertActivity } from '../../../delivery/store/features/activities/slice';
 import { savePage } from '../../store/page/actions/savePage';
 import { selectState as selectPageState, updatePage } from '../../store/page/slice';
-import ContextAwareToggle from '../Accordion/ContextAwareToggle';
 import PropertyEditor from '../PropertyEditor/PropertyEditor';
 import lessonSchema, {
   lessonUiSchema,
   transformModelToSchema as transformLessonModel,
   transformSchemaToModel as transformLessonSchema,
 } from '../PropertyEditor/schemas/lesson';
-import screenSchema, { getScreenData, screenUiSchema } from '../PropertyEditor/schemas/screen';
+import screenSchema, {
+  transformScreenModeltoSchema as transformScreenModel,
```

Comment:
dont really need to rename these, the only reason I did above is because in the lesson file I just called them `transformModelToSchema` without the word Lesson in it. (since that was in the context of the filename). but if you are going to include "Screen" in the function name then it wouldn't be ambiguous. alternatively you could rename the exported method to just be `transformModelToSchema` and then rename it here